specs/SingleAppHost.md

```diff
@@ -0,0 +1,65 @@
+# Background
+While WebViews from different app processes normally don't share the same WebView browser process instance, app processes from the same product suite may share the same webview browser process
+instance by specifying the same user data folder when creating WebView2Environment object.
+The `ExclusiveUserDataFolderAccess` property is for the developer to express the sharing intent, so that we could provide optimized security and performance according expected usage.
+
+# Description
+The `ExclusiveUserDataFolderAccess` property indicates whether other processes can create WebView2 sharing the same WebView browser process instance by using WebView2Environment created with the same user data folder.
```

Comment:
This needs to be rewritten to take the point of view of the user data folder. Right now, the text says "The exclusive data folder property indicates whether processes can be shared" - this is confusing because the name of the property appears unrelated to what it actually controls.

> The `ExclusiveUserDataFolderAccess` property specifies whether other WebView environments can be created with the same user data folder. Setting exclusive access prevents the WebView processes from being shared with those belonging to other environments because sharing occurs only between instances that use the same user data folder.